blocks/columns/columns.css

```diff
@@ -370,7 +370,7 @@
 }
 
 .biencommun .section:has(.columns.two-column) h1 {
-  color: #726248;
+  color: var(--fondations-titre-h2);
```

Comment:
I think this color has to be for biencommun - since the rule has biencommun in it